pom.xml

```diff
@@ -35,11 +35,11 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>2.13</version>
+                <version>2.17</version>
                 <configuration>
                     <configLocation>build-tools/src/main/resources/checkstyle/checkstyle.xml</configLocation>
                     <includeTestSourceDirectory>true</includeTestSourceDirectory>
-                    <suppressionsLocation>checkstyle/suppressions.xml</suppressionsLocation>
+                    <!-- <suppressionsLocation>checkstyle/suppressions.xml</suppressionsLocation> -->
```

Comment:
Just a test and even with the new strict, build failing checkstyle config, the suppressions are not necessary, but I'll leave them if we want to add to them in the future.